docs/api/vulnerability-info.md

```diff
@@ -33,6 +33,16 @@ The data file schema does not allow for modification or redaction of known vulne
 Therefore if a server's vulnerability data source (for example the [GitHub Advisories Database](https://github.com/advisories)) modifies an existing advisory, the NuGet server must modify the page that the vulnerability information was previously reported.
 One way to achieve this with the suggested partition scheme is to treat all vulnerability modifications and deletions as a trigger to regenerate the complete `base.json` file, and empty `updates.json`.
 
+## Reusing nuget.org's vulnerability data
+
+NuGet does not enforce that resources in the [service index](./service-index.md), or [the vulnerability index](#vulnerability-index), must be on the same server as the service index itself.
```

Comment:
nit: I'd flip this. NuGet does not require....

My suggestion would also be to add some explicit information that every resource is expected to be available. 
I don't know where to add that but feels important.

Reply:
Is there anyone who would assume it's ok to use URLs that aren't available?  It seems weird to explicitly write "don't do this thing that is obviously wrong".

Reply:
I was think more like available to the end user. 

Maybe this is more of a client thing. 

Basically, if you block a part of a feed, and it breaks, it's not a supported scenario.

Reply:
I added something to the "host your own feed" page.  What do you think?